cdisc_rules_engine/operations/base_operation.py

```diff
@@ -80,6 +80,13 @@ def _handle_dictionary_result(self, result):
         )
         return self.evaluation_dataset
 
+    def filter_data(self, data) -> pd.DataFrame:
```

Comment:
we use the leading underscore convention `_filter_data` to indicate that a method is protected
https://www.geeksforgeeks.org/access-modifiers-in-python-public-private-and-protected/